skmediate/conditional_independence.py

```diff
@@ -0,0 +1,85 @@
+# scikit-learn class for cross-covariance statistic
+
+import numpy as np
+from scipy import linalg
+from sklearn.linear_model import LinearRegression
+from sklearn.covariance import EmpiricalCovariance
+
+
+class ConditionalCrossCovariance(object):
+    """
+    Implements conditional dependence testing between multivariate quantities.
+
+    """
+
+    def __init__(
+        self,
+        regression_estimator=None,
+        covariance_estimator=None,
+        precision_estimator=None,
+    ):
+        """
+        Initializes ConditionalCrossCovariance with base estimators.
+
+
+        """
+
+        if regression_estimator is None:
+            regression_estimator = LinearRegression()
```

Comment:
In some cases, maybe we want a separate regression estimator for `Y ~ X` and for `Z ~ X`. For example, I can see ridge or elastic net (or, why not, SGL!) being useful for white matter features, but maybe not as useful for other kinds of things that enter here as Z.